test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -1008,6 +1026,7 @@ describe('Dropdown Component', () => {
         .simulate('blur')
         .simulate('focus')
 
+      domEvent.keyDown(document, { key: 'ArrowDown' })
```

Comment:
There is a comment on 1024 that we should update or remove:

```
// blur, focus, move item selection up
```

Since we are now opening after the focus.